tests/test_clinical_trials_data.py

```diff
@@ -41,3 +41,25 @@ def test_change_group_by_changes_data(monkeypatch):
     labels = ['Interventional', 'Observational [Patient Registry]']
     assert ctd.get_labels() == labels
     assert ctd.get_current_filters() == []
+
+
+def test_change_current_filters(monkeypatch):
```

Comment:
Might be able to break these down into smaller tests in order to make it more explicit as to what is happening